pymedphys/tests/dicom/test_dose.py

```diff
@@ -79,19 +79,25 @@ def test_require_patient_orientation_be_HFS():
 
     test_ds_dict["no orient"] = ds_no_orient
 
+    test_orientations = ("HFS", "HFP", "FFS", "FFP")
+
     for orient, ds in test_ds_dict.items():
-        if orient == "HFS":
-            require_patient_orientation_be_HFS(ds)
-
-        elif orient == "no orient":
-            with pytest.raises(AttributeError) as ea:
-                require_patient_orientation_be_HFS(ds)
-            assert "object has no attribute 'ImageOrientationPatient'" in str(ea.value)
-
-        else:
-            with pytest.raises(ValueError) as ev:
-                require_patient_orientation_be_HFS(ds)
-            assert (
-                "The supplied dataset has a patient orientation "
-                "other than head-first supine" in str(ev.value)
-            )
+
+        for test_orient in test_orientations:
+            if orient == test_orient:
+                require_patient_orientation(ds, test_orient)
+
+            elif orient == "no orient":
+                with pytest.raises(AttributeError) as ea:
+                    require_patient_orientation(ds, test_orient)
+                assert "object has no attribute 'ImageOrientationPatient'" in str(
+                    ea.value
+                )
+
+            else:
+                with pytest.raises(ValueError) as ev:
+                    require_patient_orientation(ds, test_orient)
+                assert (
+                    "The supplied dataset has a patient orientation "
+                    f"other than {test_orient}" in str(ev.value)
+                )
```

Comment:
What's your opinion here? I've never been that much of a fan directly asserting the text of a raised message... Do you think this was needed in the original test? Or do you think it may have been better just writing:

```python
with pytest.raises(ValueError) as ev:
    require_patient_orientation(ds, test_orient)
```

Reply:
I agree, I think testing that the error is raised is sufficient. The text being "correct" doesn't affect the functionality. I'll make the adjustment :)

Reply:
Thanks @pchlap